WeLibraryOS/rich-markdown-editor
# Add prop for custom handling of image uploads (handleImageUploadOverride)

## General
Being able to paste (or drag and drop) an image/file into the Editor is super cool. But sometimes you want to handle a file upload differently (e.g. if you want to store the image as an attachment rather than have it be embedded in the text).

Enter handleImageUploadOverride! A handy new prop that just passes the uploaded image/file as a parameter and lets the consuming library deal with it however it so pleases.

## To Test
Use with [wl-react#489](https://github.com/WeLibraryOS/wl-react/pull/489). When you paste images into the editor, you should see them get uploaded instead of getting embedded into the text
